### ISSUE TITLE
need help with Regression model and batch-normalization

### ISSUE THREAD

**Issue by bazimian** [Verified-Intelligence/alpha-beta-CROWN#47]
Hi,

I would like to kindly ask for help with the following two items:
1. I have trained a feed forward DNN regression model in pytorch with say 12 input neurons and 50 output neurons. I have used dropout and batch-normalization layers in between also. The structure is given below. I was wondering if the alpha-beta-CROWN module can handle such a model. If so do I need to make any changes regarding the BN layers or everything will be taken care of automatically.

dropout_rate = 0.2
number_of_neurons = 30

model = nn.Sequential(
    nn.Linear(x_train.shape[1], number_of_neurons),
    nn.ReLU(),
    nn.BatchNorm1d(number_of_neurons),
    nn.Dropout(dropout_rate),
    nn.Linear(number_of_neurons, number_of_neurons),
    nn.ReLU(),
    nn.BatchNorm1d(number_of_neurons),
    nn.Dropout(dropout_rate),
    nn.Linear(number_of_neurons, y_train.shape[1])
).cuda()

2. My main concern is with the data preparation. Currently I have one .npy file with 2k rows and 12 columns for input (X) and 2k rows and 50 columns for output (Y). I have 2k test samples. looking at the sample data given for example in mnist_eran folder, y_eran.npy only has one column which makes sense for classification problems. I am wondering what should I do to my dataset specifically for the output to make it compatible with the alpha-beta-CROWN module. Also what other changes should I make in the config.YAML file to make the analysis compatible for regression analysis.

Thank you in advance for your help!

**Comment by shizhouxing** [comment]
Explained via email earlier. It could be supported by using VNNLIB specs instead of .npy data.
